UPGRADE-3.x.md

```diff
@@ -1,6 +1,56 @@
 UPGRADE 3.x
 ===========
 
+## Deprecated unspecified default_formatter configuration node
+
+The `default_formatter` configuration node will become required.
+
+```yaml
+sonata_formatter:
+    default_formatter: my_formatter
+```
+
+## Deprecated specifying a logger in the Sonata\FormatterBundle\Formatter\Pool constructor
+
+Providing a logger to a `Sonata\FormatterBundle\Formatter\Pool` instance is
+optional, and optional dependencies should be provided through a setter.
```

Comment:
IMO this is arguable, I would just stick to facts: Logger can now be injected via setter instead of passing it to the constructor.